engine/src/main/java/org/terasology/math/MatrixUtils.java

```diff
@@ -61,14 +38,8 @@ public static FloatBuffer matrixToFloatBuffer(Matrix4fc m) {
      * @return A new FloatBuffer containing the matrix in column-major form.
      * @deprecated used JOML method that uses Matrix3fc
```

Comment:
I'm not sure about deprecating these methods, since they still add value by taking care of the buffer creation.
If anything I'd deprecate `matrixToFloatBuffer(Matrix3fc m, FloatBuffer fb)`, but they aren't currently marked for deprecation. ("they" as in both, the mat3 and mat4 variations)